src/main/flight/pid.c

```diff
@@ -191,6 +191,7 @@ PG_RESET_TEMPLATE(pidProfile_t, pidProfile,
         .fixedWingItermThrowLimit = FW_ITERM_THROW_LIMIT_DEFAULT,
         .fixedWingReferenceAirspeed = 1000,
         .fixedWingCoordinatedYawGain = 1.0f,
+        .loiter_direction = NAV_LOITER_RIGHT,
```

Comment:
Don't forget to increase the PG version. 4 to 5
PG_REGISTER_PROFILE_WITH_RESET_TEMPLATE(pidProfile_t, pidProfile, PG_PID_PROFILE, 4);

Reply:
done